### PR TITLE
add library udata_event_service to produce Kafka messages

### PR DESCRIPTION
**Pull request by sixtedemaupeou**

### CODE REVIEW

File: requirements/install.in [hunk 1]
```diff
@@ -50,6 +50,7 @@ requests==2.24.0
 simplejson==3.17.0
 StringDist==1.0.9
 tlds
+udata_event_service==0.0.7
```
Comment (maudetes) [inline requirements/install.in:53]: We could remove `kafka-python` explicit dependency in install.in now?

File: udata/core/dataset/events.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+
 from udata.utils import to_iso_datetime, get_by
 from udata.models import Dataset
 from udata.tasks import task
-from udata.event.producer import produce
+from udata.event.producer import get_topic
 from udata.event.values import KafkaMessageType
 
 
+
```
Comment (maudetes) [inline udata/core/dataset/events.py:12]:
```suggestion
```

File: udata/core/dataset/events.py [hunk 3]
```diff
@@ -1,12 +1,15 @@
 import datetime
 from flask import current_app
+from udata_event_service.producer import produce
```
Comment (maudetes) [inline udata/core/dataset/events.py:3]: I think it's not needed, except if we want to encapsulate udata_event_service everywhere, in order to do:
```
from udata.event import produce
```

**Review by maudetes — COMMENTED** [review]
Some minor comments on this PR.

We'll need to modify the message `value` in https://github.com/opendatateam/udata-search-service/pull/30 accordingly :) 

Thank you!

File: udata/core/dataset/events.py (hunk 3 above)
Comment (maudetes) [reply udata/core/dataset/events.py:3]: Ah sorry, I meant in [udata/event/__init__.py](https://github.com/opendatateam/udata/pull/2743/files#diff-f84ff018df265711184d54da187835a924f3f64d7a84352f84cbc09586a8f584)

**Comment by maudetes** [comment]
Turning this into a Draft to make sure to make it when synchronized with https://github.com/opendatateam/udata-search-service/pull/30.

File: udata/core/dataset/events.py (hunk 3 above)
Comment (sixtedemaupeou) [reply udata/core/dataset/events.py:3]: I was just replacing the previous produce but unsure whether any of the two imports of that file is actually useful.

File: udata/event/__init__.py [hunk 4]
```diff
@@ -1,2 +1,2 @@
 import udata.core.dataset.events  # noqa
-from .producer import produce, KafkaProducerSingleton # noqa
+from udata_event_service.producer import produce # noqa
```
Comment (maudetes) [inline udata/event/__init__.py:2]: It doesn't seem needed anymore
```suggestion
```

**Review by maudetes — APPROVED** [review]
I think we're all good!

Could you add an entry in the CHANGELOG.md? :pray:

**Review by maudetes — APPROVED** [review]

**Pull request merged**